pkgs/build-support/docker/default.nix

```diff
@@ -392,6 +392,7 @@ rec {
 
       postMount = ''
         echo "Packing raw image..."
+        mkdir $out
```

Comment:
```suggestion
        mkdir -p $out
```

Reply:
I usually prefer using `-p` in case there are more than one directory to create.

Reply:
Done